pkg/operator/ceph/config/monstore_test.go

```diff
@@ -72,6 +73,89 @@ func TestMonStore_Set(t *testing.T) {
 	assert.Contains(t, execedCmd, " config set mon.* unknown_setting 10 ")
 }
 
+func TestMonStore_Delete(t *testing.T) {
```

Comment:
```suggestion
func TestMonStoreDelete(t *testing.T) {
```

Reply:
I prefer to keep the same naming than other test, for ex.: TestMonStore_Set